diff --git a/ggml/src/ggml-sycl/quants.hpp b/ggml/src/ggml-sycl/quants.hpp
index 1f5b62740..48c16b9ef 100644
--- a/ggml/src/ggml-sycl/quants.hpp
+++ b/ggml/src/ggml-sycl/quants.hpp
@@ -44,7 +44,7 @@ template <> struct block_q_t<GGML_TYPE_Q4_0> {
         static constexpr uint32_t qk       = QK4_0;
         static constexpr uint32_t qi       = QI4_0;
         static constexpr uint32_t qr       = QR4_0;
-        static constexpr uint32_t vdr_mmvq = 2;
+        static constexpr uint32_t vdr_mmvq = 4;  // was 2: increased to process more blocks per subgroup, shift bottleneck from dp4a to BW
     };
 
     static constexpr std::pair<int, int> get_block_offset(const int block_index, const int /* nblocks */) {
